_articles/how-to-contribute.md

```diff
@@ -494,7 +494,7 @@ If the project is on GitHub, here's how to submit a pull request:
 * **Reference any relevant issues** or supporting documentation in your PR (for example, "Closes #37.")
 * **Include screenshots of the before and after** if your changes include differences in HTML/CSS. Drag and drop the images into the body of your pull request.
 * **Test your changes!** Run your changes against any existing tests if they exist and create new ones when needed. Whether tests exist or not, make sure your changes don't break the existing project.
-* **Contribute in the style of the project** to the best of your abilities. This may mean using indents, semi-colons or comments differently than you would in your own repository, but makes it easier for the maintainer to merge, others to understand and maintain in the future.
+* **Contribute to the style of the project** to the best of your abilities. This may mean using indents, semi-colons or comments differently than you would in your own repository, but makes it easier for the maintainer to merge, others to understand and maintain in the future.
```

Comment:
This one I prefer in its original wording. "Contribute _to_ the style" has a different meaning than "contribute _in_ the style"; the intent here is, "when making a contribution to someone else's repository, try to adhere to the conventions and aesthetics of that repository, even when you have different preferences in your own work," which the original captures a little more clearly imo.

Reply:
Yeah, **contribute to** sounds better. My bad!